File: userservice/src/main/java/com/project/userservice/model/Question.java
package com.project.userservice.model;

import lombok.Data;

@Data
public class Question 
{
    private Long questionId;

    private String quizId;

    private String question;

    private String option1;

    private String option2;

    private String option3;

    private String option4;

    private String correctAnswer;
}

File: result/model/Question.py
from dataclasses import dataclass

@dataclass
class Questions:

    questionId : int

    quizId : str

    question : str

    option1 : str

    option2 : str

    option3 : str

    option4 : str

    correctAnswer : str

    isCorrect : bool = False

    response : str = None
